src/main/java/teammates/ui/controller/InstructorCourseStudentDetailsEditSaveAction.java

```diff
@@ -57,14 +57,25 @@ public ActionResult execute() throws EntityDoesNotExistException {
             
             boolean isSectionChanged = student.isSectionChanged(originalStudentAttribute);
             boolean isTeamChanged = student.isTeamChanged(originalStudentAttribute);
+            boolean isEmailChanged = student.isEmailChanged(originalStudentAttribute);
             if (isSectionChanged) {
                 logic.validateSectionsAndTeams(Arrays.asList(student), courseId);
             } else if (isTeamChanged) {
                 logic.validateTeams(Arrays.asList(student), courseId);
             }
             
             logic.updateStudent(studentEmail, student);
-            statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_EDITED, StatusMessageColor.SUCCESS));
+            
+            boolean isSendEmail = getRequestParamAsBoolean(Const.ParamsNames.EMAIL_SEND_CHECK);
```

Comment:
Similarly, `isSendEmail` is also vague. It is specifically about session summary and thus the variable name should capture it.